### PR TITLE
dependency on more stable version of dc-js

### PR DESCRIPTION
**Pull request by gazal-k**
Just a thought. Wouldn't it be better to use a more stable version of dc-js dependency than referencing develop branch.

### CODE REVIEW

**Comment by Tim-Intellipharm** [comment]
dc.js is currently in beta for version 2 so there is not a stable version of it yet. The develop branch _should_ be the most stable version of dc.js 2.0.0.

**Comment by gazal-k** [comment]
:smiley: let me rephrase. Yes, there is no latest stable of dc-js 2 yet. And dcjs release 2.0.0-beta17 and develop branch have no differences. But I think it might be a good idea to refer to a 2.0.0 version rather than to develop branch. As soon as a stable 2.0 hits, PRs intended for 2.1 will start getting merged to develop and develop branch will have code that can be considered to be in alpha stage or earlier.

**Comment by Tim-Intellipharm** [comment]
Does `~2.0.0` work with the beta version `2.0.0-beta.17`?

**Comment by gazal-k** [comment]
Affirmative :+1:

**Comment by gazal-k** [comment]
Hold on a sec though, let me make a small change

**Comment by gazal-k** [comment]
This is the relevant line when trying bower install

``` sh
bower dcjs#~2.0.0                          resolved git://github.com/dc-js/dc.js.git#2.0.0-beta.17
```

**Comment by gazal-k** [comment]
Sorry, the new version of bower seems to have messed this up: https://github.com/bower/bower/issues/1896

**Comment by Tim-Intellipharm** [comment]
They have just release [v1.5.2](https://github.com/bower/bower/releases/tag/v1.5.2) which fixes this back up again.
